xarray/tests/test_groupby.py

```diff
@@ -2606,3 +2607,12 @@ def test_default_flox_method() -> None:
         assert kwargs["method"] == "cohorts"
     else:
         assert "method" not in kwargs
+
+
+@requires_cftime
```

Comment:
I think it may be failing because of the new warnings at the moment:
```suggestion
@requires_cftime
@pytest.mark.filterwarnings("ignore")
```